README.md

```diff
@@ -18,6 +18,7 @@ addon | version | summary
 [mrp_production_note](mrp_production_note/) | 9.0.1.0.0 | Notes in production orders
 [mrp_production_partner_note](mrp_production_partner_note/) | 9.0.1.0.0 | MRP - Partner production notes
 [mrp_sale_info](mrp_sale_info/) | 9.0.1.0.0 | Adds sale information to Manufacturing models
+[mrp_bom_version](mrp_bom_version/) | 9.0.1.0.0 | BoM versioning
```

Comment:
Don't modify this. It's done automatically and you will provoke for sure conflicts.